test/dialect/mssql/test_reflection.py

```diff
@@ -566,12 +566,157 @@ def test_indexes_with_filtered(self, metadata, connection):
 
         eq_(sorted(filtered_indexes), ["([x]='test')", "([y]>=(5))"])
 
+        t2 = Table("t", MetaData(), autoload_with=connection)
+        idx = list(sorted(t2.indexes, key=lambda idx: idx.name))[0]
+        self.assert_compile(
+            CreateIndex(idx), "CREATE NONCLUSTERED INDEX idx_x ON t (x) WHERE ([x]='test')"
```

Comment:
this was an existing test that needed to be updated to include the NONCLUSTERED keyword in the DDL statement since the reflected index now contains that info